CMakeLists.txt

```diff
@@ -7,7 +7,8 @@ set(Timetabler_VERSION_MINOR 3)
 set(Timetabler_VERSION_PATCH 0)
 set(CMAKE_BUILD_TYPE Release)
 
-set(CMAKE_CXX_FLAGS -Wno-literal-suffix)
+# set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
```

Comment:
Why have this line?

Reply:
In the latest Glucose, no-literal-suffix warnings are not present.